var/spack/repos/builtin/packages/regcm/package.py

```diff
@@ -7,7 +7,7 @@
 
 
 class Regcm(AutotoolsPackage):
-    """RegCM ICTP Regional Climate Model."""
+    '''RegCM, ICTP Regional Climate Model (https://ictp.it).'''
```

Comment:
Apologies for the OCD, but every other package uses double quotes. Can you maintain them?

Reply:
I'll fix it in a new commit.